src/main/java/edu/hawaii/its/api/controller/GroupingsRestController.java

```diff
@@ -122,6 +122,9 @@ public ResponseEntity<String> getGrouping(@RequestBody(required = true) List<Str
             @RequestParam(required = true) String sortBy,
             @RequestParam(required = true) Boolean isAscending) {
         logger.info("Entered REST getGrouping...");
+        String groupPathsStr = groupPaths != null ? groupPaths.toString() : "null";
```

Comment:
Correct me if I am wrong, you don't need to check for null or convert groupPaths to a string since the logger and String.format functions will handle those cases.